postcss.config.js

```diff
@@ -0,0 +1,15 @@
+const autoprefixer = require('autoprefixer');
+
+module.exports = {
```

Comment:
autoprefixer can be bundled in other packages and then we may end up having issues.

I would recommend creating the `browserslist` file in project root with content like

```
[production]
last 2 Chrome versions
last 2 Firefox versions
last 2 iOS versions
last 2 Safari versions
last 2 Edge versions
android >= 4.0

[development]
last 2 Chrome versions
```

Reply:
Done